relay-server/src/actors/processor.rs

```diff
@@ -2232,6 +2233,36 @@ impl EnvelopeProcessorService {
         Ok(())
     }
 
+    #[cfg(feature = "processing")]
+    fn extract_spans(&self, state: &mut ProcessEnvelopeState) {
+        // For now, drop any spans submitted by the SDK.
+        state.managed_envelope.retain_items(|item| match item.ty() {
+            ItemType::Span => ItemAction::DropSilently,
+            _ => ItemAction::Keep,
+        });
+
+        if !state
+            .project_state
+            .has_feature(Feature::ExtractStandaloneSpans)
+        {
+            return;
+        };
+
```

Comment:
Should probably check for `event.type == transaction` here.

Reply:
Line 2250 returns early if the event isn't a transaction, does that cover the need?